lib/command/src/Obelisk/Command.hs

```diff
@@ -161,6 +161,8 @@ deployInitOpts = DeployInitOpts
   <$> strArgument (action "directory" <> metavar "DEPLOYDIR" <> help "Path to a directory that it will create")
   <*> strOption (long "ssh-key" <> action "file" <> metavar "SSHKEY" <> help "Path to an ssh key that it will symlink to")
   <*> some (strOption (long "hostname" <> metavar "HOSTNAME" <> help "hostname of the deployment target"))
+  <*> strOption (long "ssl-hostname" <> metavar "SSLHOSTNAME" <> help "SSL hostname")
+  <*> strOption (long "admin-email" <> metavar "ADMINEMAIL" <> help "Site admin email")
```

Comment:
We need a better description here.  How about "Email address where administrative alerts will be sent"?